data-raw/watershed/thomes/thomes_creek.Rmd

```diff
@@ -17,13 +17,17 @@ library(cvpiaHabitat)
 None Planned 
 
 ## Instream Spawning and Rearing Habiat
-
+```{r}
+source('data-raw/watershed/regional_approximation/mean_wua_description.R')
+```
 **Data Source:** 
-No watershed specific salmonid habitat data was available for Antelope Creek. A regional weighted usable area (WUA) relationship with flow was derived for Antelope Creek by averaging the WUA values on Battle Creek, Butte Creek, Clear Creek, Cottonwood Creek and Cow Creek. The geomorphic and hydrologic conditions in Antelope Creek (watershed area = X; active channel width = Y; 2-year flow = Z) are similar to those on Battle Creek (watershed area = X; active channel width = Y; 2-year flow = Z), Butte Creek (watershed area = X; active channel width = Y; 2-year flow = Z); Clear Creek (watershed area = X; active channel width = Y; 2-year flow = Z), Cottonwood Creek (watershed area = X; active channel width = Y; 2-year flow = Z), and Cow Creek (watershed area = X; active channel width = Y; 2-year flow = Z). The regional WUA relationships for Antelope Creek were multiplied by the length of spawning and rearing extents mapped by the Science Integration Team (SIT).  
 
-### Mean WUA plot 
+`r print_mean_wua_details('Thomes Creek')`
+
+### Modeling Details 
+
+The spawning and rearing data used for Thomes Creek is the regional approximation for the Upper-mid Sacramento River Region. This regional approximation can he found [here](http://cvpia-habitat-docs-markdown.s3-website-us-west-2.amazonaws.com/watershed/Regional_Approximation.html). 
```

Comment:
replace "This regional approximation" with "More information about this methodology "